docs/apis-tools/web-modeler-api/tutorial.md

```diff
@@ -0,0 +1,247 @@
+---
+id: modeler-api-tutorial
+title: Tutorial
+sidebar_position: 3
+slug: /apis-tools/web-modeler-api/tutorial
+description: "Step through an example to create a new project, add a collaborator, view project details, and delete a project using the Web Modeler API."
+---
+
+In this tutorial, we'll step through examples to highlight the capabilities of the Web Modeler API, such as creating a new project, adding a collaborator to a project, viewing the details of a project, and deleting a project.
+
+## Prerequisites
+
+- If you haven't done so already, [create a cluster](/components/react-components/create-cluster.md).
+- Upon cluster creation, create your first client by navigating to **Console > Organization > Administration API > Create new credentials**. Ensure you determine the scoped access for client credentials. For example, in this tutorial we will create a project, add a collaborator, and delete a project. Ensure you check all the boxes for Web Modeler and Zeebe client scopes.
```

Comment:
Does this tutorial really require Zeebe client scopes?

Reply:
Adjusted in latest commit 👍